pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -98,6 +98,33 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		disableFeatureGates()
 	})
 
+	DescribeTable("when spec validator pass, should allow",
+		func(op admissionv1.Operation) {
+			ar, err := newAdmissionReviewForVMI(newBaseVmi(), op)
+			Expect(err).ToNot(HaveOccurred())
+
+			admitter := &VMICreateAdmitter{ClusterConfig: config, SpecValidators: []ValidateSpec{validateStub()}}
+			resp := admitter.Admit(context.Background(), ar)
+			Expect(resp.Allowed).To(BeTrue())
+		},
+		Entry("create request", admissionv1.Create),
+		Entry("any request", admissionv1.Operation("")),
```

Comment:
The empty operation is basically any other non create operation.
I added tests to cover it in case a special treatment for Create request will be introduced that will be coverge in tests.
Looking at this again it might not be necessary, WDYT?